pre_award/assess/assessments/templates/assessments/change_request_success_page.html

```diff
@@ -0,0 +1,52 @@
+{% extends "assess/base.html" %}
+{% from "assess/macros/sub_criteria_navbar.html" import navbar %}
+{% from "assess/macros/sub_criteria_heading.html" import sub_criteria_heading %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+
+
+{% set pageHeading -%}
+Request changes –
+{% if current_theme.id == sub_criteria.id %}
+  {{ sub_criteria.name }} – {{ sub_criteria.project_name }}
+{% else %}
+  {{ current_theme.name }} – {{ sub_criteria.name }} – {{ sub_criteria.project_name }}
+{% endif %}
+{% endset %}
+
+{% block header %}
+  {% include "assess/components/header.html" %}
+{% endblock header %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-half-from-desktop">
```

Comment:
Let's please make this `govuk-grid-column-one-third `. Although I will admit I am not too sure.

Design shows it a bit longer than 1/3 but shorted than 1/2. We used 1/3 for other success pages, so let's keep it consistent.